### ISSUE TITLE
Fixed Version Run Time bundled with application is trying to update it self

### ISSUE THREAD

**Issue by dqwork** [MicrosoftEdge/WebView2Feedback#1089]
**Description**
We have bundled the Fixed version of the run time into our application for distribution, since the installers require elevation which our app cannot request.

We've had reports from our testing team who do not have any version of the runtime installed already, that some security software is flagging a process being triggered related to edge.

We investigated and found this log file
C:\Users\<userame>\AppData\Local\Temp\MicrosoftEdgeUpdate.log

I can't attach the log file here as it potentially contains sensitive info. But some of the log entries clearly show some process trying to update the run time

`[03/18/21 14:23:14.685][MicrosoftEdgeUpdate:msedgeupdate][28400:19400][DllEntry]["**PII**\AppData\Local\Microsoft\EdgeUpdate\MicrosoftEdgeUpdate.exe" /ua /installsource chromerecovery]`

`[03/18/21 14:23:33.479][MicrosoftEdgeUpdate:msedgeupdate][8372:37016][DownloadManager::DoDownloadPackage][appid={F3C4FE00-EFD5-403B-9569-398A20F1BA4A}; version=1.3.141.63; package_name=MicrosoftEdgeUpdateSetup_X86_1.3.141.63.exe]
`

This clearly some background process trying to update the run time.. I don't know if it succeeds... but still **this should not be running at all.** If we distribute this to customers we've essentially installed something they did not agree to that starts running in the background

**Version**
SDK: 1.0.705.50
Runtime: Fixed Version 88.0.705.81
Framework: .Net Framework 4.6.2 WinForms
OS: Windows 10

**Comment by billybooth** [comment]
Following this, as we are planning on distributing the fixed version runtime until the evergreen runtime gets wider adoption, and the behavior described here would be completely unacceptable.

**Comment by dqwork** [comment]
For those that use the fixed version run time. Check this folder 
<userfolder>\AppData\Local\Microsoft\EdgeUpdate

Some part of the fixed version run time is dropping files and programs in there that start themself and try and do some kind of update process.
Our security software at work has flagged this process attempting to start is self - can anyone from Microsoft please explain whats going on here?

**Comment by champnic** [comment]
Hey all - the Fixed Version should definitely not be trying to update itself. That defeats the whole purpose!

Have you verified that any of these files are coming from the Fixed Version? Or that it's the WebView2 Runtime? Or that it updates the files in your Fixed Version deployment to the machine?

 That looks like it could be coming from the regular Microsoft Edge updater, which is being rolled out pretty broadly (as a replacement to the previous Microsoft Edge browser). It's also possibly a different WebView2 app that's installing the runtime. For example, Microsoft Office recently began deploying WebView2 Runtime: https://docs.microsoft.com/en-us/DeployOffice/webview2-install#webview2-runtime-installation

**Comment by dqwork** [comment]
@champnic thanks for getting back to me.

I'm 99% its my app that has bundled the fixed version run time.

At my company, we are very locked down, we don't take windows updates immediately and I know for a fact the new Edge and the related run time are something they are currently assessing and if there was any option to delay my company would do. 

We also have some security software that monitors are processes that start and anything not whitelisted will be blocked and flagged up.

Shortly after I installed and ran my release version app (with it all bundled up) I got a pop up saying this ms edge installer had been blocked, I potentially put it down to all the dev stuff I'd been doing. So I got my colleague who had never touched a web view runtime, to install the app and give it a go. Shortly after installing and running they got the same pop up and the same folder in their APP data folder. I can confirm with more colleagues today

**Comment by dqwork** [comment]
the security software thats flagging this, is some what lagging in the UX department, so I can't see the full path. However the file that it flags as trying to run

![image](https://user-images.githubusercontent.com/77854269/112124633-7818c800-8bba-11eb-8db9-82818c61289d.png)

somewhere within C:\Users\<user>\appdata\local\microsoft\edgeupdate\install\...

**Comment by dqwork** [comment]
we also have a log file MicrosoftEdgeUpdate.log - I'm not really comfortable uploading that to Github though as I don't know what info has been captured. Is there a way I can send that to you directly?

**Comment by champnic** [comment]
Sure - you can email it to me at champnic@microsoft.com.

It certainly sounds from the extra detail that it's related to the app being deployed. Can you share how you are deploying your app and the Fixed Version runtime? Or can you share the app itself? Or create a minimum repro app that just deploys the Fixed Version runtime in the same manner as your main app? The Fixed Version shouldn't even have MicrosoftEdgeUpdate.exe in it (which if it does would obviously be a bug we get fixed ASAP).

**Comment by dqwork** [comment]
@champnic sorry for the delay, but you know how the big companies can be with sharing some stuff with external entities. Anyway, I've sent you an email today will as many details as I could share.

For those following this issue. I installed our app that uses the fixed version binaries on a brand spanking new Windows Server 2019 VM and shortly after using the fix version web view, some kind edge updater was installed which then went on to install the evergreen time, which looks quite conclusive to me (I repeated this a couple of times with brand new servers and also a server that had been running for a day or 2
